### ISSUE TITLE
Windows dialog options do not make sense

### ISSUE THREAD

**Issue by mjainQA** [Komodo/KomodoEdit#2955]
### Short Summary

User options about when getting started tutorial already running do not make sense. See the screenshot.

### Steps to Reproduce

1. Load getting started tutorial
2. Go all the way to the last screen
3. Check open tutorial box
4. Click ok
5. Notice the notification dialog

### Expected results
1. Options in the dialog should be according to question. Not Cancel or OK.

### Platform Information

*Komodo Edit or IDE?* 
*Komodo Version?* 
*Operating System (and version)?* 
Komodo IDE, version 11.0.0-alpha1, build 90564, platform macosx.
Built on Sun Aug 27 23:39:13 2017.
### Additional Information
![image](https://user-images.githubusercontent.com/26605593/29787992-d2238304-8be5-11e7-91dd-a9aff2cb68d4.png)

**Comment by cgchoffman** [comment]
So you selected "start the getting started tutorial" option while you had it running?  I'd say this is user error no?

**Comment by cgchoffman** [comment]
Make sure you fill out the "expected result".  I don't know what the action item you're expecting is.

**Comment by Naatan** [comment]
I cannot reproduce this. Curious about Carey's question also, did you already have it open?

**Comment by mjainQA** [comment]
Yes, as mentioned in the repro steps. Make sure you have the getting started tutorial open before starting the 'Lets get started' wizard.

**Comment by Naatan** [comment]
Then this is by design.

**Comment by Naatan** [comment]
Oh my bad, the ask here is for the button text to change.
